tools/debian_buildenv.sh

```diff
@@ -100,6 +100,7 @@ case "$1" in
             qt5keychain-dev \
             qtdeclarative5-dev \
             qtscript5-dev \
+            qttools5-dev \
```

Comment:
I think we shouldn't target this to 2.4.

Reply:
2.5? main? IDC, just need to know what to target.

Reply:
I recommend main, as 2.5 is already close to release.

Reply:
rebased.